CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## master (unreleased)
 
+### New features
+
+* Make the `track-state` middleware invokeable directly, by requesting the new `"cider/state"` op.
```

Comment:
I'd name this `get-state` or even better - `get-ns-state`, so it's clearer what it does. It wasn't that important when it couldn't be triggered, but now it should be some verb in line with most ops.

Reply:
I chose it for consistency with this key https://github.com/clojure-emacs/cider/blob/eedbab26684cb1ce01a940b5b0da9ff78aeb4eac/cider-repl.el#L238 , however I'd agree that the alternatives are clearer

Reply:
Yeah, I get this, but I don't think such a name would be appropriate for something one can invoke. The response will obviously stay the same.